system-addon/lib/ActivityStream.jsm

```diff
@@ -22,13 +23,16 @@ const {TopSitesFeed} = Cu.import("resource://activity-stream/lib/TopSitesFeed.js
 const {TopStoriesFeed} = Cu.import("resource://activity-stream/lib/TopStoriesFeed.jsm", {});
 
 const REASON_ADDON_UNINSTALL = 6;
+const LOCALE = Services.locale.getRequestedLocale();
+// for now, we only want to show top stories by default to these locale
+const ACCEPTED_POCKET_LOCALES = ["en-US", "en-CA"];
 
 // Sections, keyed by section id
 const SECTIONS = new Map([
   ["topstories", {
     feed: TopStoriesFeed,
     prefTitle: "Fetches content recommendations from a configurable content provider",
-    showByDefault: false
+    showByDefault: ACCEPTED_POCKET_LOCALES.includes(LOCALE) || false
```

Comment:
Actually, just inline all of `LOCALE` and `ACCEPTED_POCKET_LOCALES` to right here to be clear this is special temporary logic for preffing on/off top stories.